### PR TITLE
Made search value persist on view change

### PR DESCRIPTION
**Pull request by Remi749**
but value disappears when user clicks on the search field #todo

### CODE REVIEW

**Comment by olemp** [comment]
Didn't work @Remi749. Seems like it worked after change in https://github.com/Puzzlepart/prosjektportalen/pull/721/commits/2b974d702cd8c2b0ebec1ea5ff576943cdc4228e.

Can you test aswell?

File: src/js/WebParts/DynamicPortfolio/index.tsx [hunk 1]
```diff
@@ -642,13 +642,8 @@ export default class DynamicPortfolio extends BaseWebPart<IDynamicPortfolioProps
...
             }
         }
-        await this.updateState(updatedState);
 
-        // Check if the searchbar had searchterm on new view
-        if (updatedState.searchTerm) {
-            document.querySelectorAll('input[id^="SearchBox"].ms-SearchBox-field')[0].attributes[4].value = this.state.searchTerm;
-            // TODO: Make the search value stay when use clicks the search bar after view change, right now it dissapears
-        }
+        await this.updateState(updatedState);
```
Comment (Remi749) [inline src/js/WebParts/DynamicPortfolio/index.tsx]: Putting the await below makes it break. It should be as before as the updateState clears the searchform.

Comment (olemp) [reply src/js/WebParts/DynamicPortfolio/index.tsx]: Have you tested after my commit?

Comment (olemp) [reply src/js/WebParts/DynamicPortfolio/index.tsx]: (Shouldn't use document.querySelectorAll code like this in a React app.)

Comment (Remi749) [reply src/js/WebParts/DynamicPortfolio/index.tsx]: > Have you tested after my commit?

Didn't see the commits you made above, I've tested it and it works. I was certain that I had already tried this and therefore ended up with the `document.querySelectorAll` method of doing it.

**Pull request merged**